src/mywxtools.h

```diff
@@ -138,8 +136,10 @@ struct ImageDropdown : wxOwnerDrawnComboBox {
     }
 
     void OnDrawItem(wxDC &dc, const wxRect &rect, int item, int flags) const {
-        auto bm = bitmaps_display[item];
-        sys->ImageDraw(bm, dc, rect.x + FromDIP(3), rect.y + FromDIP(3));
+        auto bm = bitmaps[item];
+        wxBitmap sbm;
+        ScaleBitmap(*bm, FromDIP(1.0) / dd_icon_res_scale, sbm);

```

Comment:
This change seems unrelated from the tab height, please put these in separate PRs!

Also scaling a bitmap on every render seems expensive? Why not stick this bitmap initialization in a function, and re-run it if DPI changes?

Reply:
Hello Wouter,
Thanks for your reply! I opened a separated PR for the first commit.
And also a big thanks for your input on the second commit. This seems a better way to go. I will look how I implement this.
Regards
Tobias